.github/workflows/ci.yml

```diff
@@ -26,16 +26,17 @@ jobs:
         git config --global user.email "x@y.z"
     - name: Checkout Commit
       uses: actions/checkout@v1
+    - name: checkout main
+      run: git branch -f main origin/main
+    - name: Install pnpm
+      uses: pnpm/action-setup@v2.0.1
+      with: 
+        version: next
     - name: Setup Node
       uses: actions/setup-node@v2
       with:
         node-version: ${{ matrix.node }}
-    - name: checkout main
-      run: git branch -f main origin/main
-    - name: install pnpm and npm
```

Comment:
it seems that we can replace that with: https://github.com/pnpm/action-setup

Reply:
ok but the code that you removed was also installing npm@7 globally

Reply:
It means that the npm@7 still install globally here?

Reply:
I assume some tests will fail if npm 7 won't be installed globally. Now tests don't run because no code changes were done in packages.

Reply:
OK，i have added it back.